.github/workflows/plugin-check.yml

```diff
@@ -53,4 +52,6 @@ jobs:
             exclude-directories: vendor_prefixed
             exclude-checks: |
                 trademarks
-                late_escaping
\ No newline at end of file
+                late_escaping
+                plugin_updater_detected
+                WordPress.WP.AlternativeFunctions.*
```

Comment:
Not a check name, but a PHPCS error code.

See https://github.com/swissspidy/wp-plugin-check-action for possible check names to exclude.

This one might be coming from plugin_review_phpcs.

If there‘s interest, happy to look into adding an option for ignoring specific error codes via regex/glob though!

Reply:
ah, thanks! makes sense. let me try the supported codes! I also noticed above we have plugin_updater_detected but on the action docs it is listed as [late_escaping](https://github.com/swissspidy/wp-plugin-check-action#:~:text=plugin_header_text_domain-,late_escaping,-plugin_updater), which one is correct?

Reply:
updated in https://github.com/Yoast/wordpress-seo/pull/20808/commits/b93a8f2044a8c4f9d64e0f8d8b7b6a7990450e17

Reply:
> This one might be coming from plugin_review_phpcs.

Is this information logged anywhere? it would be great to know for each error/warning, which check triggered it (ie what to exclude if you want to ignore that check)

Reply:
I don't think so. Would be a good enhancement request for https://github.com/WordPress/plugin-check